### PR TITLE
Add GetDataFromTreeToken

### PR DESCRIPTION
**Pull request by Monar**
With this token its possible to use [react-apollo-hooks](https://github.com/trojanowski/react-apollo-hooks#server-side-rendering) server side rendering. 

```
import {  GetDataFromTreeToken } from 'fusion-plugin-apollo';
import { renderToString } from 'react-dom/server';
import { getMarkupFromTree } from 'react-apollo-hooks';

const getDataFromTree = (tree) =>
  getMarkupFromTree({ renderFunction: renderToString, tree });

/* [...] */
  const app = new App(<Main />);
/* [...] */
  if(__NODE__) {
    app.register(GetDataFromTreeToken, getDataFromTree);
  }
  return app;
```

### CODE REVIEW

**Comment by Monar** [comment]
@KevinGrandon When can we expected next release with this change ?

File: fusion-plugin-apollo/src/plugin.js [hunk 1]
```diff
@@ -34,6 +35,7 @@ export type DepsType = {
   schema: typeof GraphQLSchemaToken.optional,
   endpoint: typeof GraphQLEndpointToken.optional,
   getApolloClient: typeof ApolloClientToken,
+  getDataFromTree: typeof GetDataFromTreeToken,
```
Comment (KevinGrandon) [inline fusion-plugin-apollo/src/plugin.js]: Looks like this just needs to be `typeof GetDataFromTreeToken.optional`. I'll go ahead and see if I can make that update inline.

**Comment by Monar** [comment]
@KevinGrandon hey what are the perspectives of merging this PR?

**Comment by KevinGrandon** [comment]
> @KevinGrandon hey what are the perspectives of merging this PR?

Sorry, we've had some infrastructure issues with our internal repos that we wanted to resolve with automation, but will do so manually since it shouldn't block our contributors. Let's land this, thanks a lot for the PR!

**Pull request merged**